plugins/orchestrator/tsconfig.json

```
@@ -1,7 +1,7 @@
 {
   "extends": "@backstage/cli/config/tsconfig.json",
   "include": ["src", "dev"],
-  "exclude": ["node_modules"],
+  "exclude": ["node_modules", "src/generated"],
```

```suggestion
  "exclude": ["node_modules"],
```

I don't think you need to exclude `src/generated`. e.g. https://github.com/janus-idp/backstage-plugins/blob/main/plugins/servicenow-actions/tsconfig.json#L4